### ISSUE TITLE
Implement time series resource phase 1

### ISSUE THREAD

**Issue by dtarb** [hydroshare/hydroshare#5]
This involves 

1.  [x] 1. Settle the resource content model.  This implies specification of the name for the resource type and documentation of the storage format.  The detail of the documentation is case specific.  Specify the scientific metadata.

2. [x] 2. Add capability to upload and download resource files that are already in the specified storage format through web interface.  Edit metadata through web interface.  Serialize metadata.
An assumption here is that a time series resource will involve only one content file.  The step 2 capability handles that raw file and may parse metadata out of it but does not support loading of entire resources in the bagit packaging format.

Phase 2 and beyond are not part of this step and include the following that should be incrementally added later (see issue #559)

3.  [ ] 3. Add capability to upload and download resource files already is specified storage format through API.  Set and get metadata through API.

4.  [ ] 4. Write a few lines of code to generate a preview/thumbnail image of the time series to go in the visualization folder (for display on the landing page)?  I'm thinking (Jeff) we could do something simple with Matplotlib.  It wouldn't be dynamic, just a static plot image that gets generated when somebody uploads a time series resource

5.  [ ] 5. Visualization tool

6.  [ ] 6. Tools for importing from other formats (e.g. CSV, Excel)

6.  [ ] 7.  Tools for exporting to other formats (e.g. CSV)

6.  [ ] 8.  Tools for establishing a web service endpoint to serve this resource content (WaterML flavors)

6.  [ ] 9.  Tool to import resource packages as a complete resource data model bag.  This involves a reflector function to generate the metadata in Django from the XML format metadata and will support direct exchange of this resource type with iRODS and with other clients.

**Comment by pkdash** [comment]
@horsburgh and @dtarb  - I think this issue should be closed.

**Comment by horsburgh** [comment]
@pkdash - in the time since we created Phase 1 of time series, we made some modifications to the ODM2 schema.  I think we should verify that HydroShare time series work with the most recent ODM2 schema in SQLite before we close this.  I can create an example ODM2 database using the latest schema for you to test.  Let me know when you are ready.

**Comment by pkdash** [comment]
@horsburgh We should get it done for next release (1.6). Let me know when you have an example ODM2 database for me to work with.

**Comment by pkdash** [comment]
@horsburgh - If you want you can test this fix at dev.hydroshare.org

**Comment by horsburgh** [comment]
@pkdash - I just tested on Dev.  It looks good.  Only one caveat and that is that in my source ODM2 database, the Spatial Reference for the latitude and longitude coordinates for the monitoring site is listed as "Unknown" (it gets set that way because it isn't in the WaterML 1.1 file I'm loading into the ODM2 database).  So, in the resource that is created, the Coordinate System/Geographic Projection in the Coverage element is set to "Unknown".  I think this is probably the desired behavior.  However, given that we have decided that coordinates expressed in the Dublin Core Coverage elements should be in WGS84, we may want to flag this as not having a valid coordinate system (e.g., not accept "Unknown" as a valid coordinate system). 

I'm going to open a new issue for the Phase 2 time series stuff.
